api/v1/bestie_types.go

```diff
@@ -28,10 +28,16 @@ type BestieSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster.
 	// Important: Run "make" to regenerate code after modifying this file.
 
+	// The size of the deployment
 	Size    int32  `json:"size"`
+
```

Comment:
File is not `goimports`-ed with -local github.com/opdev/l5-operator-demo/l5-operator (goimports)
Is this part of the solution?

Reply:
no its the inter complaining i thought i fixed it..